lib/beaker/dsl/install_utils/windows_utils.rb

```diff
@@ -5,14 +5,15 @@ module InstallUtils
       # This module contains methods useful for Windows installs
       #
       module WindowsUtils
-
         # Given a host, returns it's system TEMP path
+        #
         # @param [Host] host An object implementing {Beaker::Hosts}'s interface.
-        def get_temp_path(host)
-          # under CYGWIN %TEMP% may not be set
-          tmp_path = on(host, Command.new('ECHO %SYSTEMROOT%', [], { :cmdexe => true }))
-          tmp_path.output.gsub(/\n/, '') + '\\TEMP'
+        #
+        # @return [String] system temp path
+        def get_system_temp_path(host)
```

Comment:
To avoid the new alias, wonder if it might be better to just change the signature of `get_temp_path` and pass in a new `kind` parameter with default value like:

```ruby
def get_temp_path(host, kind = :system)
```